test/InjectorTest.php

```diff
@@ -325,13 +328,13 @@ public function testComplexDeepDependencyConfiguration()
 
         $config = new Config([
             'types' => [
-                TreeTestAsset\Level2::class => [
+                TreeTestAsset\Level2::class           => [
                     'parameters' => [
                         'opt' => $expected1,
                     ],
                 ],
-                'Level2.Alias' => [
-                    'typeOf' => TreeTestAsset\Level2::class,
+                'Level2.Alias'                        => [
```

Comment:
This looks lie auto fixer. @geerteltink and @michalbundyra  This looks really wired and is hard to read for multi dimensional arrays. Is this the intended effect/style?

Reply:
As explained [here](https://github.com/laminas/laminas-di/pull/16#discussion_r499811386), we should keep this as is as it will probably re-formatted again by the beautifier.